### PR TITLE
Adjust tasit-sdk test suite to work against testnet

### LINKED ISSUE #135: Work on the deployment and make sure we know what contracts we’ll point to and on what testnet

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#135 

### Auto-close the issue?

Closes #135 

### Types of changes

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### CODE REVIEW

File: packages/tasit-action/src/testHelpers/helpers.js [hunk 1]
```diff
@@ -95,6 +97,9 @@ const mineOneBlock = async provider => {
 };
 
 const mineBlocks = async (provider, n) => {
+  // Do nothing if provider isn't a JSON-RPC
```
Comment (pcowgill) [inline packages/tasit-action/src/testHelpers/helpers.js]: What's an example of such a provider?

File: packages/tasit-contract-based-account/src/GnosisSafe.test.js [hunk 2]
```diff
@@ -41,13 +41,18 @@ describe("GnosisSafe", () => {
   beforeEach("", async () => {
     const { address: someoneAddress } = someone;
 
-    confirmEtherBalances(provider, [GNOSIS_SAFE_ADDRESS], [ZERO]);
-    confirmBalances(erc20, [GNOSIS_SAFE_ADDRESS], [ZERO]);
-    confirmBalances(nft, [GNOSIS_SAFE_ADDRESS], [ZERO]);
+    // Expect an already funded Gnosis Safe wallet
```
Comment (pcowgill) [inline packages/tasit-contract-based-account/src/GnosisSafe.test.js]:
```suggestion
    // Expect an already-funded Gnosis Safe wallet
```

File: packages/tasit-contract-based-account/src/GnosisSafe.test.js [hunk 3]
```diff
@@ -62,11 +67,6 @@ describe("GnosisSafe", () => {
   });
 
   describe("test cases that need ETH deposit to the contract-based account", async () => {
-    beforeEach("faucet", async () => {
-      await etherFaucet(provider, minter, GNOSIS_SAFE_ADDRESS, ONE);
```
Comment (pcowgill) [inline packages/tasit-contract-based-account/src/GnosisSafe.test.js]: So this wasn't necessary?

File: packages/tasit-sdk/src/Decentraland.test.js [hunk 4]
```diff
@@ -110,12 +110,19 @@ describe("Decentraland", () => {
...
       });
 
-      beforeEach("", async () => {
+      beforeEach("check Gnosis Safe balance", async () => {
         gnosisSafe = new GnosisSafe(GNOSIS_SAFE_ADDRESS);
 
-        // Expect an already-funded Gnosis Safe wallet
-        await confirmEtherBalances(provider, [GNOSIS_SAFE_ADDRESS], [ONE]);
-        await confirmBalances(mana, [GNOSIS_SAFE_ADDRESS], [ONE_HUNDRED]);
+        // Expect an already funded Gnosis Safe wallet
```
Comment (pcowgill) [inline packages/tasit-sdk/src/Decentraland.test.js]:
```suggestion
        // Expect an already-funded Gnosis Safe wallet
```

File: packages/tasit-sdk/src/Decentraland.test.js [hunk 5]
```diff
@@ -110,12 +110,19 @@ describe("Decentraland", () => {
...
+        let etherBalance = await provider.getBalance(GNOSIS_SAFE_ADDRESS);
+        etherBalance = bigNumberify(etherBalance);
+        const one = bigNumberify(ONE);
+        expect(etherBalance.gte(one)).to.be.true;
+
+        let manaBalance = await mana.balanceOf(GNOSIS_SAFE_ADDRESS);
```
Comment (pcowgill) [inline packages/tasit-sdk/src/Decentraland.test.js]: Why did you move away from the confirmBalances strategy here? Because of the `greater than or equal to`? We could have a separate convenience function for that or an extra param for the existing function if you want.

File: packages/tasit-action/src/testHelpers/helpers.js (hunk 1 above)
Comment (marcelomorgado) [reply packages/tasit-action/src/testHelpers/helpers.js]: Infura uses RPC calls over HTTP instead of the RPC protocol directly.

File: packages/tasit-contract-based-account/src/GnosisSafe.test.js (hunk 3 above)
Comment (marcelomorgado) [reply packages/tasit-contract-based-account/src/GnosisSafe.test.js]: Because thart expectation is being already made by a `beforeEach` hook before that:     // Expect an already funded Gnosis Safe wallet

Comment (marcelomorgado) [reply packages/tasit-contract-based-account/src/GnosisSafe.test.js]: And the Gnosis Safe wallet is now funded by the script that runs after migrations and before all test suites.

File: packages/tasit-sdk/src/Decentraland.test.js (hunk 5 above)
Comment (marcelomorgado) [reply packages/tasit-sdk/src/Decentraland.test.js]: Yes, sure!

Comment (marcelomorgado) [reply packages/tasit-sdk/src/Decentraland.test.js]: Done.

Comment (pcowgill) [reply packages/tasit-sdk/src/Decentraland.test.js]: Thanks! Resolving

File: packages/tasit-contract-based-account/src/GnosisSafe.test.js (hunk 3 above)
Comment (pcowgill) [reply packages/tasit-contract-based-account/src/GnosisSafe.test.js]: Ah, got it. Thanks!

Comment (pcowgill) [reply packages/tasit-contract-based-account/src/GnosisSafe.test.js]: Resolving this

File: packages/tasit-action/src/testHelpers/helpers.js (hunk 1 above)
Comment (pcowgill) [reply packages/tasit-action/src/testHelpers/helpers.js]: Ohhh, interesting. So maybe that's worth mentioning specifically in the comment. I assume it's the same for Etherscan and Alchemy too.

Comment (pcowgill) [reply packages/tasit-action/src/testHelpers/helpers.js]:
```suggestion
  // Do nothing if provider isn't a JSON-RPC
  // (Infura uses RPC calls over HTTP as opposed to JSON-RPC directly)
```

File: packages/tasit-action/src/erc721/ERC721Full.test.js [hunk 6]
```diff
@@ -154,7 +154,7 @@ describe("TasitAction.ERC721.ERC721Full", () => {
       expect(event.to).to.equal(bob.address);
       expect(event.tokenId.toNumber()).to.equal(tokenId);
 
-      await confirmBalances(erc721, [ana.address, bob.address], [0, 1]);
+      await tokenBalancesAreEqual(erc721, [ana.address, bob.address], [0, 1]);
```
Comment (pcowgill) [inline packages/tasit-action/src/erc721/ERC721Full.test.js]: I won't have this gate merging, but the function name as it's worded right now kind of makes it sound like Ana's and Bob's addresses should be equal

Comment (pcowgill) [reply packages/tasit-action/src/erc721/ERC721Full.test.js]: Maybe `expectExactBalances` and `expectMinimumBalances` would work instead?

Comment (pcowgill) [reply packages/tasit-action/src/erc721/ERC721Full.test.js]: `expect` as opposed to `confirm` since `confirm` is a loaded word in the Ethereum ecosystem

Comment (marcelomorgado) [reply packages/tasit-action/src/erc721/ERC721Full.test.js]: I liked `expectExactBalances` and `expectMinimumBalances`, I'll rename these.

Comment (pcowgill) [reply packages/tasit-action/src/erc721/ERC721Full.test.js]: Great, thanks

Comment (pcowgill) [reply packages/tasit-action/src/erc721/ERC721Full.test.js]: Resolving this since it changed in https://github.com/tasitlabs/TasitSDK/pull/281
